.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Feature request
+about: Suggest a new feature for Weave Flux
+title: ''
+labels: [blocked-needs-validation, enhancement]
+assignees: ''
+
+---
+
+**Describe the feature**
+A clear and concise description of the feature.
+
+**What would the new user story look like?**
+How would the new interaction with Flux look like?
+0. What are the prerequisites for this?
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
```

Comment:
No one wants to see an error for a feature request :))

Reply:
That's what happens if you do this and haven't had :coffee: at the office yet. The queue this morning was already too long. :-)